tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -336,7 +336,14 @@ public async Task AnonymousJobsCanBeExecutedMultipleTimes()
     {
         var fakeTimer = new FakeTimeProvider();
         ServiceCollection.AddSingleton<TimeProvider>(fakeTimer);
-        ServiceCollection.AddNCronJob(n => n.AddJob(async (ChannelWriter<object> writer) => await writer.WriteAsync(true), "* * * * *"));
+        async Task WriteTrueAsync(ChannelWriter<object> writer, CancellationToken ct)
+        {
+            await Task.Delay(10, ct);
```

Comment:
Does it need that delay?

Reply:
Without it the Mac test fails sometimes. I think it may have to do with running these Jobs too quickly. The 10ms simulates a tiny bit of work without bogging down the unit tests. It seems like it could have something to do with the FakeTimeProvider

Reply:
Interesting - recently MS shifted to MacOS ARM64 for the GH runners. We could also try to take the x64 version instead. But that just hides the issue.